### ISSUE TITLE
Clicking on support button generates a warning in console 

### ISSUE THREAD

**Issue by Timunas** [spotify/backstage#1498]
Whenever a click is performed on a support button a warning is shown the console as shown below:

![console_warning](https://user-images.githubusercontent.com/18316742/86181340-bd293580-bb25-11ea-9fe7-c379b725a196.png)

Although I can't see a clear problem in the UI point of view.

## Expected Behavior

No console warnings expected.

## Current Behavior

Console logs are shown in the console. Although I can't see a clear problem in the UI side.

## Possible Solution

Change the SupportButton ListemItemText elements by disabling the default typography usage.

```
            <ListItem>
              <ListItemIcon>
                <GroupIcon />
              </ListItemIcon>
              <ListItemText
                disableTypography
                primary={<Typography>Contact</Typography>}
                secondary={
                  <div>
                    {contactEmails.map((em, index) => (
                      <Typography key={index}>
                        <Link>{em}</Link>
                      </Typography>
                    ))}
                  </div>
                }
              />
            </ListItem>
```

## Steps to Reproduce

1. git clone
2. yarn install
3. yarn start
4. Open browser dev tools
5. Click support button in home page for example

## Context

It seems to be a known issue for Typography component of Material UI.

[Stackoverflow](https://stackoverflow.com/questions/41928567/div-cannot-appear-as-a-descendant-of-p)
[Material UI code](https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/Typography/Typography.js#L140)

## Your Environment

- NodeJS Version (v12): v12.18.2
- Operating System and Version (e.g. Ubuntu 14.04): Zorin OS 12.4
- Browser Information: Firefox 77.0.1

**Comment by Timunas** [comment]
Hi @Rugvip I would like to help fixing this issue. Is it ok? Are there any guidelines I should follow?

**Comment by Rugvip** [comment]
@Timunas Thanks, sure! For this one it's prolly enough to make sure you're formatting the code with prettier.

**Comment by govindgoel** [comment]
Can I work on this issue @Timunas

**Comment by Timunas** [comment]
Hi @govindgoel  I'm going to open the PR in a few minutes.

**Comment by stefanalund** [comment]
Fixed in #1530
